addons/binding/org.openhab.binding.russound/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,324 @@
+            <channel id="turnonvolume" typeId="turnonvolume"/>
+            <channel id="donotdisturb" typeId="string"><label>Do Not Disturb</label><description>Do Not Disturb</description></channel>
+            <channel id="partymode" typeId="string"><label>Party Mode</label><description>Party Mode</description></channel>
+            <channel id="status" typeId="button"><label>Status</label><description>Zone Status</description></channel>
+            <channel id="volume" typeId="volume"/>
+            <channel id="mute" typeId="button"><label>Mute</label><description>Mute</description></channel>
+            <channel id="page" typeId="button"><label>Page</label><description>Page Status</description></channel>
+            <channel id="sharedsource" typeId="button"><label>Shared Source</label><description>Shared Source Status</description></channel>
+            <channel id="sleeptimeremaining" typeId="sleeptime"/>
+            <channel id="lasterror" typeId="readonlystring"><label>Last Error</label><description>Last Zone Error</description></channel>
+            <channel id="enabled" typeId="button"><label>Enabled</label><description>Whether the zone is enabled or not</description></channel>
+            <channel id="repeat" typeId="button"><label>Repeat</label><description>Toggle the repeat mode</description></channel>
+            <channel id="shuffle" typeId="button"><label>Shuffle</label><description>Toggle the shuffle mode</description></channel>
+            <channel id="rating" typeId="button"><label>Rating</label><description>How to rate current song</description></channel>
+            <channel id="keypress" typeId="string"><label>KeyPress event</label><description>Send a KeyPress event</description></channel>
+            <channel id="keyrelease" typeId="string"><label>KeyRelease event</label><description>Send a KeyRelease event</description></channel>
+            <channel id="keyhold" typeId="string"><label>KeyHold event</label><description>Send a KeyHold event</description></channel>
+            <channel id="keycode" typeId="string"><label>KeyCode event</label><description>Send a KeyCode event</description></channel>
+            <channel id="event" typeId="string"><label>Generic event</label><description>Send a generic event</description></channel>
+        </channels>
+
+        <config-description>
+            <parameter name="zone" type="integer" min="1" max="8" required="true">
+                <label>Zone ID</label>
+                <description>The zone identifier</description>
+            </parameter>
+        </config-description>
+    </bridge-type>
+
+    <bridge-type id="source">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="rio"/>
+        </supported-bridge-type-refs>
+        
+        <label>Russound Source</label>
+        <description>Source (tuner, streamer, etc) within the Russound System</description>
+        
+        <channels>

```

Comment:
make sure that only a few are regular channels and the majority are advanced channels.